src/icesat2_tracks/analysis_db/B02_make_spectra_gFT.py

```diff
@@ -1,32 +1,40 @@
+import sys
+
+
 """
 This file open a ICEsat2 track applied filters and corections and returns smoothed photon heights on a regular grid in an .nc file.
 This is python 3
 """
 
-import copy
-import datetime
-import h5py
-import time
-import sys
+from icesat2_tracks.config.IceSAT2_startup import (
+    mconfig,
+    plt,
+)
```

Comment:
If we are going to explicitly list deps in files, we should probably get rid of imports like this one.
```suggestion
import matplotlib.pyplot as plt
from icesat2_tracks.config.IceSAT2_startup import mconfig
```